example/javascriptlib/publishing/2-realistic/build.mill

```diff
@@ -0,0 +1,124 @@
+package build
+
+import mill._, javascriptlib._
+
+object foo extends TypeScriptModule {
+  def generatedSources = Task {
+    os.write(
+      Task.dest / "foo.generated.ts",
+      s"""export default class FooGen {
+         |  static value: number = 123
+         |}
+      """.stripMargin
+    )
```

Comment:
We don't need to demonstrate `generatedSources` in every submodule in this file, just once is enough